README.md

```diff
@@ -283,6 +204,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly fix the link to your License page, when I click this is broken

Reply:
Issues resolved